core-web/apps/dotcms-ui/src/app/view/components/dot-toolbar/components/dot-toolbar-user/dot-toolbar-user.component.html

```diff
@@ -1,80 +1,11 @@
-    inverted="true"
-    position="right"
-    *ngIf="auth.user"
+<dot-gravatar
+    class="gravatar-button"
     id="dot-toolbar-user"
-    (wasOpen)="iframeOverlayService.show()"
-    (shutdown)="iframeOverlayService.hide()"
->
-    <div class="toolbar-user__content">
-        <header class="toolbar-user__header">
-            <dot-gravatar
-                class="toolbar-user__header-gravatar"
-                size="40"
-                [email]="
-                    auth.loginAsUser === null
-                        ? auth.user.emailAddress
-                        : auth.loginAsUser.emailAddress
-                "
-            ></dot-gravatar>
-            <div class="toolbar-user__header-info">
-                <h5 class="toolbar-user__user-name" id="dot-toolbar-user-name">
-                    {{
-                        auth.loginAsUser
-                            ? auth.loginAsUser.name || auth.loginAsUser.fullName
-                            : auth.user.name || auth.user.fullName
-                    }}
-                </h5>
-                <p class="toolbar-user__email" id="dot-toolbar-email">
-                    {{ auth.loginAsUser ? auth.loginAsUser.emailAddress : auth.user.emailAddress }}
-                </p>
-            </div>
-        </header>
-        <ul class="toolbar-user__menu" id="dot-toolbar-user-menu">
-            <li class="user-menu__item">
-                <a
-                    href="#"
-                    (click)="toggleMyAccount()"
-                    *ngIf="!auth.loginAsUser"
-                    id="dot-toolbar-user-link-my-account"
-                    [textContent]="'my-account' | dm"
-                ></a>
-            </li>
-            <li
-                data-testId="login-as"
-                class="user-menu__item"
-                *ngIf="!auth.loginAsUser && (this.haveLoginAsPermission$ | async)"
-            >
-                <a
-                    href="#"
-                    (click)="tooggleLoginAs($event)"
-                    id="dot-toolbar-user-link-login-as"
-                    [textContent]="'login-as' | dm"
-                    >login-as</a
-                >
-            </li>
-            <li class="user-menu__item">
-                <a
-                    [href]="logoutUrl"
-                    *ngIf="!auth.loginAsUser"
-                    id="dot-toolbar-user-link-logout"
-                    [textContent]="'Logout' | dm"
-                ></a>
-            </li>
-            <li class="user-menu__item">
-                <a
-                    href="#"
-                    (click)="logoutAs($event)"
-                    *ngIf="auth.loginAsUser"
-                    id="dot-toolbar-user-link-logout-as"
-                    [textContent]="'logout-as' | dm"
-                    >logout-as</a
-                >
-            </li>
-        </ul>
-    </div>
-</dot-dropdown-component>
+    [email]="getEmailFromAuthState()"
```

Comment:
Don't call methods in templates: https://medium.com/showpad-engineering/why-you-should-never-use-function-calls-in-angular-template-expressions-e1a50f9c0496

Reply:
Alright! I'm fixing it